TombEngine/Game/control/control.cpp

```diff
@@ -397,6 +397,7 @@ void KillMoveEffects()
 	ItemNewRoomNo = 0;
 }
 
+// NOTE: No one should use this ever again.
```

Comment:
Maybe it is better to rename it to: "TODO: Remove this since it is useless and shouldn't be used" or something equivalent?